### DISCUSSION TITLE
Doesn’t handle the stream

### DISCUSSION THREAD

**Discussion by krolaper** [ultralytics/yolov5#10488] [General]
At the moment, it does not process the stream. There is a launch of everything as planned, but in response to endless processing without the result and without errors.
The code was not changed. Example link: https://youtu.be/X3Iwe_d5AVk

Log:
```
detect: weights=yolov5s.pt, source=data/images, data=data/coco128.yaml, imgsz=[640, 640], conf_thres=0.25, iou_thres=0.45, max_det=1000, device=, view_img=False, save_txt=False, save_conf=False, save_crop=False, nosave=False, classes=None, agnostic_nms=False, augment=False, visualize=False, update=False, project=runs/detect, name=exp, exist_ok=False, line_thickness=3, hide_labels=False, hide_conf=False, half=False, dnn=False, vid_stride=1
YOLOv5 🚀 detect_model Python-3.10.6 torch-1.13.0+cu117 CUDA:0 (NVIDIA GeForce MX130, 2004MiB)
Fusing layers... 
YOLOv5m summary: 290 layers, 22323858 parameters, 0 gradients
```

is there a problem with some library on the server?

**Comment by glenn-jocher** [comment]
@krolaper works for me in our Colab notebook. You might just have some environment issues. 

<img width="1418" alt="Screenshot 2022-12-17 at 12 30 43" src="https://user-images.githubusercontent.com/26833433/208239623-2bd5eb14-0d27-4eec-b2ab-b3a655ce473e.png">


## Environments

YOLOv5 may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://bit.ly/yolov5-paperspace-notebook"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"></a> <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://github.com/ultralytics/yolov5/wiki/GCP-Quickstart)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://github.com/ultralytics/yolov5/wiki/AWS-Quickstart)
- **Docker Image**. See [Docker Quickstart Guide](https://github.com/ultralytics/yolov5/wiki/Docker-Quickstart) <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>
